src/css/base.scss

```diff
@@ -2453,6 +2488,14 @@ textarea,
 
 // Padding in pixels
 
+.p-0-imp {
+    padding: 0px !important;
+}
+
+.pl-0-imp {
```

Comment:
Please remove all of this, It's already available in formula.scss

Reply:
done